test/extended/util/client.go

```diff
@@ -717,6 +718,10 @@ func (c *CLI) AdminConfigClient() configv1client.Interface {
 	)
 }
 
+func (c *CLI) AdminGatewayapiClient() gatewayapiv1client.Interface {
```

Comment:
Since we don't use the wrapper defined here, we could revert all changes in this file.

Reply:
done